src/client/browser/index.js

```diff
@@ -40,11 +41,40 @@ export function startHeartbeat (heartbeatUrl, createXHR) {
     window.setInterval(() => sendXHR(heartbeatUrl, createXHR), HEARTBEAT_INTERVAL);
 }
 
+function executeInitScript (initScriptUrl, createXHR) {
+    if (!allowInitScriptExecution)
+        return;
+
+    sendXHR(initScriptUrl, createXHR)
+        .then(res => {
+            if (res.code) {
+                debugger;
```

Comment:
[here] https://github.com/DevExpress/testcafe/blob/master/Gulpfile.js#L150 the answer.
Add `src/client/browser/**/*.js` there